frontend/src/components/review/ExistingComment.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useContext } from "react";
+import { useMutation } from "@apollo/client";
+import { Flex, Button } from "@chakra-ui/react";
+import AuthContext from "../../contexts/AuthContext";
+import WIPComment from "./WIPComment";
+import {
+  UPDATE_COMMENT_BY_ID,
+  UpdateCommentResponse,
+} from "../../APIClients/mutations/CommentMutations";
+
+export type ExistingCommentProps = {
+  id: number;
+  resolved: boolean;
+  content: string;
+  time: string;
+  commentStoryTranslationContentId: number;
+  setCommentLine: (line: number) => void;
+  lineIndex: number;
+};
+
+const ExistingComment = ({
+  id,
+  resolved,
+  content,
+  time,
+  commentStoryTranslationContentId,
+  setCommentLine,
+  lineIndex,
+}: ExistingCommentProps) => {
+  const handleError = (errorMessage: string) => {
+    // eslint-disable-next-line no-alert
+    alert(errorMessage);
+  };
+
+  const [reply, setReply] = useState(false);
+
+  const { authenticatedUser } = useContext(AuthContext);
+  const [updateComment] = useMutation<{
+    updateCommentById: UpdateCommentResponse;
+  }>(UPDATE_COMMENT_BY_ID);
+
+  const resolveExistingComment = async () => {
+    try {
+      if (!resolved) {
+        const commentData = {
+          id,
+          resolved: true,
+          content,
+        };
+
+        await updateComment({
+          variables: { commentData },
+        });
+      }
+    } catch (err) {
+      handleError(err ?? "Error occurred, please try again.");
+    }
+  };
+
+  const name = `
+    ${authenticatedUser!!.firstName}
+    ${authenticatedUser!!.lastName}`;
+
+  return (
+    <Flex
+      border="1px solid"
+      borderColor="blue.50"
+      padding="14px 15px"
+      direction="column"
+      style={{
+        borderRadius: 8,
+        width: "320px",
+      }}
+      backgroundColor="white"
+    >
+      <b>Line {lineIndex}</b>
+      <Flex justify="space-between">
+        <p>{name}</p>
+        <p>{time}</p>
```

Comment:
fyi: comments panel is not fetching time right now in `CommentsQueries.ts`
```
const COMMENT_FIELDS = `
    id
    commentIndex
    content
    storyTranslationContentId
    `;
```
Are you testing this component by like so in the comments panel?
```
const CommentsList = () => {
         ....
          <ExistingComment
              key={comment.id}
              id={comment.id}
              resolved={comment.resolved}
              content={comment.content}
              time={comment.time}
              commentStoryTranslationContentId={
                commentStoryTranslationContentId
              }
              setCommentLine={setCommentLine}
              lineIndex={comment.commentIndex}
            />
          ))}
```
comments panel looks like this for me rn (time is missing)
![image](https://user-images.githubusercontent.com/15113249/134542852-057f8812-b3ba-40e7-a0dd-c1179c2800cf.png)
Based on the mocks, the time should also be on the same line as the name. Feel free to wrap these two in a `Flex` and `justifyContent: space-between` or something along those lines